### ISSUE TITLE
Error while downloading

### ISSUE THREAD

**Issue by groksrc** [NativeScript/sidekick-feedback#305]
### Did you verify whether this issue has already been reported here?
Yes

### Tell us about the problem
Apologies if this is not the right forum for this problem, but I didn't get a reply in the Slack channel. When I download NativeScript Sidekick from https://www.nativescript.org/nativescript-sidekick or https://docs.nativescript.org/sidekick/intro/installation the download completes, but then Chrome blocks the download. 

![screen shot 2018-11-30 at 11 29 19 am](https://user-images.githubusercontent.com/9099311/49305636-5fd68980-f495-11e8-83a6-70ddbee0c099.png)

### Which platform(s) does the issue occur on?
macOS 10.13.6
Chrome Version 70.0.3538.110 (Official Build) (64-bit)

### Tell us how to recreate the issue in as much detail as possible
1. Open Chrome 70
2. Go to https://www.nativescript.org/nativescript-sidekick and download for macOS.

### Does this issue happen every time?
Yes

### Send the client logs
See screenshot

**Comment by ggarabedian** [comment]
Hello @groksrc,

We tried to reproduce the reported issue on several different macOS machines with the latest version of Chrome (70.0.3538.110 - Official Build - 64-bit ), but unfortunately without success. In our tests, Chrome completed the download and didn't report the file as malicious. 

Having said that, can you please tell me if you are able to download other **.dmg** files (not from the App Store)? Additionally, if a co-worker with the same setup close to you, can you please try to download the Sidekick package on his machine?

**Comment by groksrc** [comment]
Hi @ggarabedian,

I have just tried the repro again and for whatever reason the download works normally today. It may have been a transient error or something to do with a CDN somewhere?

I worked around this issue at the time by just using Safari to complete the download. The file was downloaded and installed without issue so this was definitely something specific to Chrome. Unfortunately I don't have any other guidance to offer on what may have been the cause.

At this point I'd consider the issue as resolved since it seems to have cleared up on its own. Thanks for looking into it!
